finite-element/test/t120-poissonksp.sh

```diff
@@ -1,56 +0,0 @@
-#!/bin/sh
```

Comment:
The output of my KSP was slightly different - same information organized differently, so I temporarily pulled the file. I'll put it back.